test/uk/gov/hmrc/gform/models/JsonParseTestGroup.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.models
+
+import cats.data.NonEmptyList
+import org.scalatest.{ EitherValues, FlatSpec, Matchers }
+import play.api.data.validation.ValidationError
+import play.api.libs.json._
+
+class JsonParseTestGroup extends FlatSpec with Matchers with EitherValues with JsResultMatcher {
+
+  val jsonStr =
+    """
+      |        {
+      |          "type": "group",
+      |          "id": "gid",
+      |          "label": "glabel",
+      |          "fields": [
+      |            {
+      |              "type": "choice",
+      |              "id": "cid",
+      |              "label": "clabel",
+      |              "choices": [
+      |                "A",
+      |                "B"
+      |              ]
+      |            }
+      |          ]
+      |        }
+    """.stripMargin
```

Comment:
Since this is string which is going to be parsed we don't really need this margin pipes. As advantage of it we could use `alt+enter -> inject language reference -> json` in intellij and have syntax highlighting like this:

![image](https://cloud.githubusercontent.com/assets/2970040/26576958/9588bc70-4522-11e7-86ed-b7d44a7a5d6e.png)